### PR TITLE
Portfolio

### PR DESCRIPTION
**Pull request by simrankaur20**
Week 1 Project : Portfolio
UI LINK : https://drive.google.com/file/d/1g8IsLWN9cP0cjFWqYBOPsyQ4R0FRmhfT/view?usp=sharing

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -1,18 +1,270 @@
...
+				</p>
+
+			</div>
+		</div>
+	</div>
+
+
+	<!-- jumbotron3 start -->
+	<div class="jumbotron myjumbo" id="skills">
+		<h1 class="display-4">Skills</h1>
```
Comment (cpaliwal) [inline portfolio/src/main/webapp/index.html:169]: 'Skills' and 'Project' headings are styled differently if it's intentional then it's fine

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -1,18 +1,270 @@
...
+				<img src="images/webdev.jpeg" class="card-img-top" alt="...">
+				<div class="card-body">
+					<h5 class="card-title">Website Development</h5>
+					<p class="card-text">I know Full Stack Web Development. For frontend, I know HTML, CSS JavaScript ,
+						ReactJS. For Backend, I know NodeJS. I have worked with MongoDB and Express </p>
+
+				</div>
+			</div>
+			<div class="card">
+				<img src="images/program.jpg" class="card-img-top" alt="...">
```
Comment (cpaliwal) [inline portfolio/src/main/webapp/index.html:181]: images in skill row are of different height, if it is designed that way then it's fine

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -1,18 +1,270 @@
...
+				<li class="nav-item active">
+					<a class="nav-link" href="#community" id="colorchoice">Community</a>
+				</li>
+				<li class="nav-item active">
+					<a class="nav-link" href="#skills" id="colorchoice">Skills</a>
+				</li>
+				<li class="nav-item active">
+					<a class="nav-link" href="#projects" id="colorchoice">Projects</a>
+				</li>
+				<li class="nav-item dropdown">
```
Comment (cpaliwal) [inline portfolio/src/main/webapp/index.html:53]: I'm assuming you'll this functionality in next pull request

**Review by cpaliwal — APPROVED** [review]

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (simrankaur20) [reply portfolio/src/main/webapp/index.html:169]: yes

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (simrankaur20) [reply portfolio/src/main/webapp/index.html:53]: Actually I had forgotten to remove this line of code

**Pull request merged**